src/gui/dialog/LatexSettingsPanel.cpp

```diff
@@ -4,19 +4,39 @@
 #include <variant>
 
 #include "control/latex/LatexGenerator.h"
+#include "control/settings/Settings.h"
 
 #include "PathUtil.h"
 #include "filesystem.h"
 #include "i18n.h"
 
-LatexSettingsPanel::LatexSettingsPanel(GladeSearchpath* gladeSearchPath):
+#ifdef HAVE_GTK_SOURCEVIEW_LIB
+#include <gtksourceview/gtksource.h>
+#endif
+
+LatexSettingsPanel::LatexSettingsPanel(GladeSearchpath* gladeSearchPath, Settings* settings):
         GladeGui(gladeSearchPath, "latexSettings.glade", "latexSettingsPanel"),
         cbAutoDepCheck(GTK_TOGGLE_BUTTON(this->get("latexSettingsRunCheck"))),
-        globalTemplateChooser(GTK_FILE_CHOOSER(this->get("latexSettingsTemplateFile"))) {
+        globalTemplateChooser(GTK_FILE_CHOOSER(this->get("latexSettingsTemplateFile"))),
+        globalSettings(settings) {
```

Comment:
I don't like this... `LatexSettingsPanel` should only know about `LaTeX`-specific settings. 

Although it makes sense to include the `SourceView` settings in the `LatexSettingsPanel` (as that's the only place it's used), this should probably be changed.

Reply:
Let's keep it simple for now and have it in the `LatexSettingsPanel`. We can make it global when we need it.

Reply:
Done!